.github/workflows/ubuntu-build-intree.yml

```
@@ -0,0 +1,102 @@
+name: Build in-tree
```

```suggestion
name: In-tree build
```
(to align with the other action)